main.py

```diff
@@ -1,10 +1,65 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
+def create_race(race: dict) -> None:
+    if not Race.objects.filter(name=race["name"]).exists():
+        Race.objects.create(
+            name=race["name"],
+            description=race["description"]
+        )
+
+
+def get_race_id(race: dict) -> int:
```

Comment:
bad func naming and type annotation 
are u sure that get method returns an int ?